src/routes/auth/guards/auth.guard.spec.ts

```diff
@@ -50,6 +51,7 @@ describe('AuthGuard', () => {
         ConfigurationModule.register(testConfiguration),
         CacheModule,
         AuthRepositoryModule,
+        TestNetworkModule,
```

Comment:
As above regarding owner checking requirements.

Reply:
This shouldn't be needed also.

Reply:
Removed in 72f78b8985d92a7279ab99aafc3de01e84ecfdd7.